Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Sysconf");
```

Comment:
Hmm, I wonder if we should rename the GC setting too... I don't think we have a GameCube section to put it in, but maybe we can call it something like GameCubeLanguage? "Selected" is rather redundant, and we should make it clear that this affects GameCube games, not Wii games or the GUI.